roadmap.md

```diff
@@ -0,0 +1,87 @@
+.NET Core Roadmap
+=================
+
+.NET Core is a general purpose, modular, cross-platform and open source implementation of .NET. The .NET Core team (anyone contributing to .NET Core) aspires to enable it to work for many interesting scenarios. Some of those are already part-way to completion. Nice work, team! 
+
+This roadmap is intended to communicate the team's priorities for evolving and extending the scope of .NET Core.
+
+Microsoft is very invested in .NET Core and makes multiple "official Microsoft" .NET Core distributions available to its customers. Sweet! It is expected (and desired) that other groups (corporations or otherwise) will also become significantly invested in .NET Core, possibly making other distros available. As a result, this roadmap is written from a neutral perspective, not a Microsoft one (unless specifically called out). This document is therefore naturally open to PRs.
```

Comment:
I'm sure we'd do that if that definition would already exist :-). We're still trying to figure out exactly what this would look like (NuGet metapackage, ZIP file, some mapping, etc).

For now, the closes thing we have is UWP and ASP.NET 4 which essentially provide a list of packages that we consider "the recommended" set of packages. Essentially, the notion of a distribution is implicit there; there is no explicit concept that models that (yet).